phases/08-generative-ai/14-evaluation-fid-clip-score/code/main.py

```
@@ -0,0 +1,146 @@
+            factor = A[r][col]
+            for j in range(2 * n):
+                A[r][j] -= factor * A[col][j]
+    return [row[n:] for row in A]
+
+
+def fid(real_features, gen_features):
+    mu_r = mean_vec(real_features)
+    mu_g = mean_vec(gen_features)
+    cov_r = covariance(real_features, mu_r)
+    cov_g = covariance(gen_features, mu_g)
+    mean_sq = sum((a - b) ** 2 for a, b in zip(mu_r, mu_g))
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)